src/AutoRest.CSharp/Common/Input/InputTypes/InputOperation.cs

```diff
@@ -27,7 +27,8 @@ internal record InputOperation(
     OperationLongRunning? LongRunning,
     OperationPaging? Paging,
     bool GenerateProtocolMethod,
-    bool GenerateConvenienceMethod)
+    bool GenerateConvenienceMethod,
+    bool GroupParameters)
```

Comment:
Should we change this to something else? We already have another thing called `Grouped` or something. This might introduce ambiguity.

Reply:
Changed to `HasPropertyBag`